dev-libs/atk/atk-2.23.4.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="GTK+ & GNOME Accessibility Toolkit"
+DESCRIPTION="Provides the set of accessibility interface that are implemented \
+by other toolkits and applications.  Using the ATK interfaces, accessibility \
+tools have full access to view and control running applications."
+HOMEPAGE="https://developer.gnome.org/atk/"
+COPYRIGHT="2000-2016 fixme"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://ftp.gnome.org/pub/GNOME/sources/atk/2.23/atk-2.23.4.tar.xz"
+CHECKSUM_SHA256="6888f4e902fc4727b1687ff512d5d087050f4c0d11035566484988cf698c2c1a"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 ?x86"
+
+PATCHES="atk-$portVersion.patch"
+
+PROVIDES="
+	atk$secondaryArchSuffix=$portVersion
+	lib:libatk_1.0=$portVersion
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:gtk_doc$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	cmd:gettext$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libglib_2.0$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	runConfigure ./configure
+	make
+}
+
+INSTALL()
+{
+	make install
+	
+	rm $libDir/libatk-1.0.la
+	
+	fixPkgconfig
+	
+}
+
+TEST()
+{
+	make check
+	cd tests
+	teststateset
+	testrelation
+	testrole
+	testvalue
+	testdocument
```

Comment:
I think it would be better to prepend a "./" in front of each test\*, just in case we drop "." from PATH. BTW, I don't know why we have "." in PATH by default...